src/containers/User/Login.tsx

```diff
@@ -73,7 +73,7 @@ const Login: React.FC = () => {
 
 export default Login
 
-function hashPassword(password: string) {
+export function hashPassword(password: string) {
```

Comment:
I wouldn't bother with this, since password hashing will be done on the Server, as per #149.

Reply:
Ok, I'll remove that